benchmarks/inc/udt.hpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#pragma once
+
+template <typename Contained>
+struct trivial {

```

Comment:
Technically, this type is an aggregate. It is trivial only if `Contained` is a trivial type.